skopt/tree_opt.py

```diff
@@ -209,6 +263,21 @@ def forest_minimize(func, dimensions, base_estimator='et', n_calls=100,
     * `n_points` [int, default=1000]:
         Number of points to sample when minimizing the acquisition function.
 
+    * `x0` [list or list of lists or None]:
+        List of initial input points (if it is a list of lists)
```

Comment:
Can you organize these into points?

Similar to what is being done here (https://github.com/scikit-optimize/scikit-optimize/blob/master/skopt/space.py#L150)